opensrp-chw-core/src/main/java/org/smartregister/chw/core/utils/StockUsageReportUtils.java

```diff
@@ -0,0 +1,198 @@
+package org.smartregister.chw.core.utils;
+
+import org.joda.time.LocalDate;
+import org.smartregister.chw.core.R;
+import org.smartregister.chw.core.application.CoreChwApplication;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class StockUsageReportUtils {
+    public Map<Integer, Integer> previousMonths() {
+        Map<Integer, Integer> monthsAndYearsList = new LinkedHashMap<>();
+
+        for (int i = 0; i < 12; i++) {
+            LocalDate prevDate = new LocalDate().minusMonths(i);
+            int month = prevDate.getMonthOfYear();
+            int year = prevDate.getYear();
+            monthsAndYearsList.put(month, year);
+        }
+        return monthsAndYearsList;
+    }
+
+    public String monthNumber(String month) {
+        String valMonth = "";
```

Comment:
- Can this method name be more descriptive? It might be hard distinguishing between this method and if a variable was named `monthNumber` 
Also maybe the name should suggest it returns a number _String_ and not an actual number